app/controllers/posts_controller.rb

```diff
@@ -0,0 +1,11 @@
+class PostsController < ApplicationController
+  def index
+    # @user = User.find(params[:user_id])
+    # @posts = @user.posts
+  end
+
+  def show
+    # @user = User.find(params[:user_id])
+    # @post = @user.posts.find(params[:id])
+  end
```

Comment:
- [OPTIONAL] Please kindly remove commented code from your code base as this makes your code base dirty and less readable.